src/unitxt/metrics.py

```diff
@@ -1835,6 +1835,120 @@ def compute(
         return self.pipe(inputs, batch_size=self.batch_size)
 
 
+class LlamaIndexCorrectness(InstanceMetric):
+    """Custom metric class for evaluating correctness using Llama Index.
+
+    Attributes:
+        reduction_map (dict): A dictionary specifying the reduction method for the metric.
```

Comment:
I don't think we need the "generic" metric documentation.  I think it should be specific for LlamaIndex (e.g. which environment variables need to be set, which models are available)